src/main/kotlin/org/rucca/cheese/task/TaskMembershipService.kt

```
@@ -159,20 +278,213 @@ class TaskMembershipService(
+                                        ),
+                                    encrypted = true,
+                                )
+
+                            // Add to the list of team members' real name info
+                            teamMembersRealNameInfo.add(
+                                TeamMemberRealNameInfo(
+                                    memberId = teamMember.user.id,
+                                    realNameInfo = memberInfo,
+                                )
+                            )
+                        } catch (e: NotFoundError) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)